### PR TITLE
Add Climbing subsystem

### PR DESCRIPTION
**Pull request by aidenpike**
Adds code for traversing bars

### CODE REVIEW

File: src/main/java/frc/robot/Robot.java [hunk 1]
```diff
@@ -35,9 +35,25 @@
...
     
-    driver = new LoggableController("Driver", 0);
+    climber = new LoggableController("Climber", 0);
     operator = new LoggableController("Operator", 1);
 
+    if(this.climberEnabled) {
+      System.out.println("Initializing climber...");
+      ClimbingSensors touchA = new ClimbingSensors("TouchA", 0, 1);
+      ClimbingSensors touchB = new ClimbingSensors("TouchB", 0, 1);
+      ClimbingSensors touchC = new ClimbingSensors("TouchC", 0, 1);
```
Comment (ultragamer1010) [inline src/main/java/frc/robot/Robot.java:45]: Change IDs on B to 2, 3 and IDs on C to 4, 5

File: src/main/java/frc/robot/Robot.java [hunk 2]
```diff
@@ -35,9 +35,25 @@
   public void robotInit() {
     pdp = new LoggablePowerDistribution(1, ModuleType.kRev);
     
-    driver = new LoggableController("Driver", 0);
+    climber = new LoggableController("Climber", 0);
     operator = new LoggableController("Operator", 1);
 
+    if(this.climberEnabled) {
```
Comment (ultragamer1010) [inline src/main/java/frc/robot/Robot.java:41]: Declare this boolean

File: src/main/java/frc/robot/Robot.java [hunk 3]
```diff
@@ -35,9 +35,25 @@
...
+      System.out.println("Initializing climber...");
+      ClimbingSensors touchA = new ClimbingSensors("TouchA", 0, 1);
+      ClimbingSensors touchB = new ClimbingSensors("TouchB", 0, 1);
+      ClimbingSensors touchC = new ClimbingSensors("TouchC", 0, 1);
+      climb = new ClimbingSensors(touchA, touchB, touchC);
```
Comment (ultragamer1010) [inline src/main/java/frc/robot/Robot.java:46]: climb object should be Climbing class and have (9, 10, 2, 3, 4, 5, touchA, touchB, touchC)

**Comment by theblindbandet** [comment]
Stephen told me to, so blame him @dracco1993

**Pull request merged**